reconcile/jira_permissions_validator.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import sys
+
+from reconcile import queries
+from reconcile.status import ExitCodes
+from reconcile.utils.jira_client import JiraClient
+
+QONTRACT_INTEGRATION = "jira-permissions-validator"
+
+
+def run(dry_run: bool) -> None:
+    settings = queries.get_app_interface_settings()
+    error = False
+    for jira_board in queries.get_jira_boards(with_slack=False):
```

Comment:
Same: can we use typed queries? (Though here i understand why youd want to reuse some)